test/check-application

```diff
@@ -31,21 +31,51 @@ class TestNavigator(testlib.MachineCase):
 
         self.login_and_go("/navigator")
         # expected heading
-        b.wait_text(".pf-c-card__title", "Directories & files")
+        b.wait_text("#navigator-card-header", "Directories & files")
+        b.wait_in_text("#sidebar-card-header", "admin")
 
         # new files are auto-detected
-        m.execute("touch /home/admin/newfile")
+        m.execute("touch -t 9407031305 /home/admin/newfile")
         b.wait_visible("[data-item='newfile']")
 
         # new directories are auto-detected
-        m.execute("mkdir /home/admin/newdir")
+        m.execute("mkdir /home/admin/newdir; touch -t 9407031500 /home/admin/newfile /home/admin/newdir")
         b.wait_visible("[data-item='newdir']")
 
         # hidden files are not displayed
         m.execute("touch /home/admin/.hiddenfile /home/admin/not-hidden")
         b.wait_visible("[data-item='not-hidden']")
         b.wait_not_present("[data-item='.hiddenfile']")
 
+        # file sidebar information
+        b.click("button:contains('newfile')")
+        b.wait_text("#sidebar-card-header", "newfile")
+        b.wait_text("#description-list-owner dd", "root")
+        b.wait_text("#description-list-group dd", "root")
+        b.wait_text("#description-list-size dd", "0 B")
+        # Packit test VMs have different timezone
+        b.wait(lambda: b.text("#description-list-last-modified dd") in ["Jul 3, 1994, 5:00 PM", "Jul 3, 1994, 3:00 PM"])
+
+        # saving a file updates sidebar info
+        # FIXME: Size does not update
+        m.execute("head -c 7 /dev/zero > /home/admin/newfile")
+        b.wait_text("#description-list-size dd", "7 B")
+        b.wait_not_in_text("#description-list-last-modified ", "Jul 3, 1994")
+        m.execute("touch -t 9407031500 /home/admin/newfile")
+        b.wait(lambda: b.text("#description-list-last-modified dd") in ["Jul 3, 1994, 5:00 PM", "Jul 3, 1994, 3:00 PM"])
+
+        # clicking empty space resets sidebar
+        b.click("#folder-view")
+        b.wait_in_text("#sidebar-card-header", "admin")
+
+        # folder information doesn't contain size
+        b.click("button:contains('newdir')")
+        b.wait_text("#sidebar-card-header", "newdir")
+        b.wait_text("#description-list-owner dd", "root")
+        b.wait_text("#description-list-group dd", "root")
+        b.wait_not_present("#description-list-size")
+        b.wait(lambda: b.text("#description-list-last-modified dd") in ["Jul 3, 1994, 5:00 PM", "Jul 3, 1994, 3:00 PM"])
```

Comment:
@martinpitt any idea how to fix this properly? I get the 5:00 PM on our test VMs but 3:00 PM on packit. And setting timezone to UTC in the start of the test did not seem to help either.

Reply:
See below in my review.